### PR TITLE
Clean up of sprite helper functions

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
Cleaned up sprite helper functions.
We only really need one! ```PIXI.Sprite.from```
Added ability to pass texture options
Brings function inline with ```PIXI.Texture.from```
Also you can now pass a texture in to the from function. Handy!

### CODE REVIEW

File: packages/sprite/src/Sprite.js [hunk 1]
```diff
@@ -590,3 +554,10 @@ export default class Sprite extends Container
         }
     }
 }
+
+Sprite.fromImage = Sprite.from;
+Sprite.fromSVG = Sprite.from;
+Sprite.fromCanvas = Sprite.from;
+Sprite.fromVideo = Sprite.from;
+Sprite.fromFrame = Sprite.from;
```
Comment (bigtimebuddy) [inline packages/sprite/src/Sprite.js]: We should definitely deprecate these. Particularly converting `fromImage` and `fromSVG` function signatures to match `from(source, options)`. @GoodBoyDigital, do you want me to take care of the deprecations in this PR?

Comment (ivanpopelyshev) [reply packages/sprite/src/Sprite.js]: Lets bury the stewardess

Comment (GoodBoyDigital) [reply packages/sprite/src/Sprite.js]: Hey @bigtimebuddy if you could that would be great! I think they also exist like that in the Texture class too.

**Comment by bigtimebuddy** [comment]
@GoodBoyDigital could you take a look at this?

### Deprecated

* `PIXI.Sprite.fromImage` moves to `PIXI.Sprite.from`
* `PIXI.Sprite.fromSVG` moves to `PIXI.Sprite.from`
* `PIXI.Sprite.fromCanvas` moves to `PIXI.Sprite.from`
* `PIXI.Sprite.fromVideo` moves to `PIXI.Sprite.from`
* `PIXI.Sprite.fromFrame` moves to `PIXI.Sprite.from`
* `PIXI.Texture.fromImage` moves to `PIXI.Texture.from`
* `PIXI.Texture.fromSVG` moves to `PIXI.Texture.from`
* `PIXI.Texture.fromCanvas` moves to `PIXI.Texture.from`
* `PIXI.Texture.fromVideo` moves to `PIXI.Texture.from`
* `PIXI.Texture.fromFrame` moves to `PIXI.Texture.from`

**Pull request merged**